### ISSUE TITLE
Unity Hub on Jetson Nano 

### ISSUE THREAD

**Issue by pankhurivanjani** [org-arl/r2c2#25]
The current setup for Unity-ros2 has only one thing missing, Unity. Before installing it I searched about it and found this forum link

[Unity3d build on Jetson Nano doesn't run
](https://answers.unity.com/questions/1625790/unity3d-build-on-jetson-nano-doesnt-run.html)

This is another architecture issue.  

The question here is now do we really need unity on jetson or can we have use some hacks for it?

**Comment by ngyewch** [comment]
You don't need Unity3D on Jetson Nano. No need to run a simulator on the actual vehicle, that kinda misses the point. Also, I don't think Unity3D has an ARM build, and it's probably too heavy to run on Jetson Nano.

**Comment by pankhurivanjani** [comment]
Yeah, that is exactly what I thought, we can discuss how to use the setup without unity on jetson.

**Comment by tbkoay** [comment]
Get unityROS2 docker installed in a host (laptop/PC) in the same subnet. Install the rest of the ROS2 stacks in Jetson nano. This is supposed to be the beauty of ROS2?
